linux_docker_resources/Dockerfile

```diff
@@ -65,9 +65,22 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
   pybind11-dev \
   pydocstyle \
   python3-argcomplete \
+  python3-colcon-argcomplete \
+  python3-colcon-common-extensions \
```

Comment:
Nowhere in ci.ros2.org do we use `colcon-common-extensions`. I'd feel much better if we continued to call out exactly the extensions we want to use in this context.

Reply:
All right, b7cc5de switches to call them out individually.